components/device-mgt/org.wso2.carbon.device.mgt.common/src/main/java/org/wso2/carbon/device/mgt/common/EnrolmentInfo.java

```diff
@@ -20,7 +20,11 @@
 
 import java.io.Serializable;
 
-public class EnrolmentInfo implements Serializable {
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+public class EnrolmentInfo implements Serializable{
```

Comment:
Fixed in https://github.com/wso2/carbon-device-mgt/commit/e1cd4b1907f419f974545fc214c58cd9f247f2b2